apps/condo/domains/property/hooks/useImporterFunctions.tsx

```diff
@@ -14,12 +14,16 @@ import { Property } from '@condo/domains/property/utils/clientSchema'
 import { searchProperty } from '@condo/domains/ticket/utils/clientSchema/search'
 
 
-const createPropertyUnitsMap = (units, sections, floors) => {
+const createPropertyUnitsMap = (units, sections, floors, minFloor) => {
     const unitsOnFloor = Math.ceil(units / (floors * sections))
     if (!unitsOnFloor) {
         return
     }
 
+    if (!minFloor) {
+        minFloor = 1
+    }
+
```

Comment:
You can set it in arguments https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters

Reply:
I I knew about this, but the TC complains that I am passing a pure number type when it could be a string or Date type